fabric/bin/lib/common_ledger.sh

```diff
@@ -39,7 +39,7 @@ ledger_precond_check() {
 define_common_vars() {
     # use our wrapper for commands, so provide convience functions ..
     # note: as we might have defined FABRIC_CFG_PATH in this script, we have the pass it along!
-    PEER_CMD="env FABRIC_CFG_PATH=${FABRIC_CFG_PATH} ${FABRIC_SCRIPTDIR}/peer.sh"
+    PEER_CMD="env FABRIC_CFG_PATH=${FABRIC_CFG_PATH} CORE_PEER_GOSSIP_EXTERNALENDPOINT=127.0.0.1:7051 ${FABRIC_SCRIPTDIR}/peer.sh"
```

Comment:
strange that you would need it for client-sdk. From core.yml this seems to be required for org-external inter-peer communication which doesn't seem to be case here?    Independent of that, might be better to define this also in `core.yaml` (including comment why needed) so we have all these FPC integration-test specific stuff in a mostly single place?

Reply:
it seems the client sdk relies on the discovery service. Without this external endpoint setting, it seems that the client cannot discover the peer. I moved this configuration from `common_ledger.sh` to `core.yaml` in `integration/config.yaml` as suggested